about.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wellness Advocate Convention- About</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400&family=Montserrat:wght@800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="styles/desktop.css">
+</head>
+<body>
+    <header>
+        <ul class="social-media block">
+            <li><a href="#"><img src="assets/facebook.png" alt="facebook logo"></a></li>
+            <li><a href="#"><img src="assets/twitter.png" alt="twitter logo"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">MyPage</a></li>
+            <li><a href="#">Log out</a></li>
+        </ul>
+        <nav class="nav">
+            <div class="nav-button">
+                <span class="ham"></span>
+                <span class="ham"></span>
+                <span class="ham"></span>
+            </div>
+           <a href="index.html"><img src="assets/wellness_advocate_logo.svg" alt="wellness advocate logo" class="logo"></a> 
+            <ul class="menu-list">
+                <li class="active navMenu"><a href="index.html">Home</a></li>
+                <li class="navMenu"><a href="#program">Program</a></li>
+                <li class="navMenu"><a href="#">Join</a></li>
+                <li class="navMenu"><a href="#sponsor">Sponsor</a></li>
+                <li class="navMenu last"><a href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero about">
+            <video src="assets/bg2.mp4" class="bg-video" autoplay muted loop></video>
+            <h1 class="heading">Mindful Wellness Retreat 2023</h1>
+            <div class="event-description">
+                <p > Mindful wellness retreat aim to provide individuals with an opportunity to prioritize their well-being,   recharge, and cultivate mindfulness in a supportive and serene environment. Mindful wellness retreats typically involve a combination of  activities such as meditation, yoga, mindfulness practices, nature walks, healthy meals, and various holistic therapies.</p>
+                <p>If you want a break from the demands and stressors of daily life, please join us.</p>
+            </div>
+            <div>
+                <p>For further information contact us:</p>
+                <a class="contact" href="#">wellness-retreat-japan@ccjapan.org</a>
+            </div>
+            
+        </section>
```

Comment:
- [ ] 🙏Please ensure that you use the correct font size, font family, and color as specified in the original design. This will help your project maintain the same cool🌹 aesthetic as the original design.

Your design:
![image](https://github.com/Kccendana/capstsone-project/assets/63445083/0ad4c843-9f79-42cd-bf5d-3f5b76bc0594)

Original design:
![image](https://github.com/Kccendana/capstsone-project/assets/63445083/293ffa0c-9cfd-454b-87cb-21d62ddc5c40)